misc/snapshotter/snapshotter.sh

```diff
@@ -41,9 +49,65 @@ function fs_driver_handler() {
     COMMANDLINE+=" --config ${SNAPSHOTTER_CONFIG}"
 }
 
+function configure_snapshotter() {
+
+    echo "configuring snapshotter"
+    if [ "${CONTAINER_RUNTIME}" != "containerd" ]; then
+        die "not supported container runtime: ${CONTAINER_RUNTIME}"
+    fi
+
+    # Copy the container runtime config to a backup
+    cp "$CONTAINER_RUNTIME_CONFIG" "$CONTAINER_RUNTIME_CONFIG".bak.nydus
+
+
+    # When trying to edit the config file that is mounted by docker with `sed -i`, the error would happend:
+    # sed: cannot rename /etc/containerd/config.tomlpmdkIP: Device or resource busy  
+    # The reason is that `sed`` with option `-i` creates new file, and then replaces the old file with the new one, 
+    # which definitely will change the file inode. But the file is mounted by docker, which means we are not allowed to 
+    # change its inode from within docker container.
+    # 
+    # So we copy the original file to a backup, make changes to the backup, and then overwrite the original file with the backup.
+    cp "$CONTAINER_RUNTIME_CONFIG" "$CONTAINER_RUNTIME_CONFIG".bak
+    # Check and add nydus proxy plugin in the config
+    if grep -q '\[proxy_plugins.nydus\]' "$CONTAINER_RUNTIME_CONFIG".bak; then
+        echo "the config has configured the nydus proxy plugin!"
+    else
+        echo "Not found nydus proxy plugin!"
+        cat <<EOF >>"$CONTAINER_RUNTIME_CONFIG".bak
+        
+[proxy_plugins]
```

Comment:
Suppose that my configuration.toml has:
```
....
[proxy_plugins]
  [proxy_plugins.blah]
...
```

This change will change the file to something like:
```
....
[proxy_plugins]
  [proxy_plugins.blah]
...
  [proxy_plugins.nydus]
        type = "snapshot"
        address = ...
```

Is it a problem? I mean, will containerd be able to recognize both plugins (`nydus` and `blah`)?